src/main/java/org/checkerframework/checker/crypto/CryptoAnnotatedTypeFactory.java

```diff
@@ -90,5 +171,19 @@ private boolean compareAllowedAlgorithmOrProviderTypes(
                     AnnotationUtils.getElementValueArray(subtype, "value", String.class, true);
             return supertypeRegexList.containsAll(subtypeRegexList);
         }
+
+        /**
+         * Creates a Crypto AnnotationMirror with new values.
+         *
+         * @param annoClass classes of annotations, only classes of AllowedAlgorithms and
```

Comment:
Why the plural `classes`? It's one single class.

Reply:
Thanks, fixed on my local repo.